awx/main/models/organization.py

```diff
@@ -247,6 +248,42 @@ def __unicode__(self):
         return self.key
 
 
+class UserSessionMembership(BaseModel):
+    '''
+    A lookup table for session membership given user.
+    '''
+
+    class Meta:
+        app_label = 'main'
+
+    user = models.ForeignKey(
+        'auth.User', related_name='+', blank=False, null=False, on_delete=models.CASCADE
+    )
+    session = models.OneToOneField(
+        Session, related_name='+', blank=False, null=False, on_delete=models.CASCADE
+    )
+    created = models.DateTimeField(default=None, editable=False)
+
+    @staticmethod
```

Comment:
Typical Django pattern would be to implement a custom `Manager` or `QuerySet` subclass for these kinds of operations instead of class or static methods on a `Model`.  [A Manager is the interface through which database query operations are provided to Django models.](https://docs.djangoproject.com/en/1.11/topics/db/managers/)